client/scss/base/_typography.scss

```diff
@@ -52,3 +52,7 @@ li {
     margin-bottom: 0;
   }
 }
+
+hr {
```

Comment:
I take your point. It's a typographic element of sorts, though. Can't think of a better place for it.